### ISSUE TITLE
Packer 1.3.1 freezes when waiting on gather facts with ansible remote provisioner

### ISSUE THREAD

**Issue by pjgoodall** [hashicorp/packer#5740]
## Workstation Configuration

```bash

> lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description: Ubuntu 16.04.3 LTS
Release: 16.04
Codename: xenial

> packer --version
1.1.3

> ansible --version
ansible 2.4.2.0

> lxd --version
2.21

```
N.B. Ansible was installed using pip

## Problem
I am trying to provision an lxd container template on Ubuntu using packer using the lxd builder with the ansible remote provisioner.  The provisioning freezes when ansible is trying to gather facts. To me, this looks like a regression of https://github.com/hashicorp/packer/issues/5155
[source files -> packer-ansible-lxd.zip](https://github.com/hashicorp/packer/files/1592592/packer-ansible-lxd.zip)
[gist with output of packer -debug](https://gist.github.com/pjgoodall/2f86828e7c6f35d8e35077da5749cd2c)

**Comment by Joeskyyy** [comment]
This appears to still be an issue. Interestingly enough, it works for me with packer 1.3.3 and ansible 2.7 but it does *not* work for me with ansible2.8 at play packer 1.3.3 or 1.4.3

**Comment by SwampDragons** [comment]
Hi! I think this is a duplicate of #7667 -- go ahead and follow that issue for workarounds and further updates.

**Comment by SwampDragons** [comment]
Rather -- the original issue probably wasn't a duplicate of 7667, but I believe that yours, @Joeskyyy , is a duplicate.

**Comment by eniac111** [comment]
I can confirm the same bug with Packer 1.5.1 and Ansible 2.9.3 using LXD builder.
